app/src/main/res/values/strings.xml

```diff
@@ -166,7 +166,7 @@
     <string name="name">Name</string>
     <string name="date">Date</string>
     <string name="size">Size</string>
-    <string name="preferences_reset_notice">This will apply initial settings</string>
+    <string name="preferences_reset_notice">Applies initial settings</string>
```

Comment:
I am not sure if this gives the same vibe as a warning. Here it sounds like a description.

![Screenshot_20210828-160744_TShot_DEBUG](https://user-images.githubusercontent.com/10258162/131218975-d16e004a-d9fd-4ba3-b518-ba97fed3296a.png)

Reply:
```suggestion
    <string name="preferences_reset_notice">(Applies initial settings)</string>
```

Reply:
Can we come up with something else? I don't see a need for parentheses here.